js/src/ScatterGL.ts

```diff
@@ -674,7 +668,7 @@ export class ScatterGL extends Mark {
   }
 
   update_attribute(
-    name: string,
+    name: String,
```

Comment:
```suggestion
    name: string,
```

(Applies multiple places in the file)